### ISSUE TITLE
AssertionError: assert False

### ISSUE THREAD

**Issue by Dongzhikang** [MzeroMiko/VMamba#30]
When I ran the command pip install . && pytest, I got the following error. Does it make sense?

================================================================================================================= short test summary info ==================================================================================================================
FAILED test_selective_scan.py::test_selective_scan[1-1-False-False-False-4096-itype1-wtype0] - AssertionError: assert False
FAILED test_selective_scan.py::test_selective_scan[1-2-False-False-False-4096-itype1-wtype0] - AssertionError: assert False
FAILED test_selective_scan.py::test_selective_scan[2-1-False-False-False-4096-itype1-wtype0] - AssertionError: assert False
FAILED test_selective_scan.py::test_selective_scan[2-2-False-False-False-4096-itype1-wtype0] - AssertionError: assert False
FAILED test_selective_scan.py::test_selective_scan[3-1-False-False-False-4096-itype1-wtype0] - AssertionError: assert False
FAILED test_selective_scan.py::test_selective_scan[3-2-False-False-False-4096-itype1-wtype0] - AssertionError: assert False
FAILED test_selective_scan.py::test_selective_scan[4-1-False-False-False-4096-itype1-wtype0] - AssertionError: assert False
FAILED test_selective_scan.py::test_selective_scan[4-2-False-False-False-4096-itype1-wtype0] - AssertionError: assert False
======================================================================================================= 8 failed, 1912 passed in 1503.00s (0:25:03) ========================================================================================================

**Comment by MzeroMiko** [comment]
It is supposed to be unusual.  

And In our model, we only use `itype0 (float32)`, `has_delta_bias=True` and `delta_softplus=True` so it doesn't matter.

I also encountered this failure when I try to merge `the nrow feature` in `fwd` and `bwd` into the `mamba repo`.  In default testing in their repo, the parameters are fixed into [`itype=torch.float32,has_delta_bias=True, delta_softplus=True,...`](https://github.com/state-spaces/mamba/blob/main/tests/ops/test_selective_scan.py), so I did not notice these mistakes before. But when I unlocked them, I found same mistakes in `mamba_ssm-1.1.3.post1+cu122torch2.2cxx11abiFALSE-cp310-cp310-linux_x86_64.whl` with cuda12.

By the way, in this repo, we did not add `nrow feature of bwd` into `selective scan`, due to those existing failures.
